tutorials/gSlack/index.md

```diff
@@ -0,0 +1,172 @@
+# gSlack
```

Comment:
Add a preamble, similar to the [other tutorials](https://github.com/GoogleCloudPlatform/community/blob/master/tutorials/styleguide.md)

```
---
title: Google Cloud Platform Community Tutorial Style Guide
description: Learn about proper style for writing Google Cloud Platform community tutorials.
author: jimtravis
tags: Tutorial, Write, Styleguide
date_published: 2017-03-03
---
```

The template will automatically turn the title into a top-level header.